bundles/pom.xml

```diff
@@ -66,7 +66,6 @@
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>rat-maven-plugin</artifactId>
-                <version>1.0-alpha-2</version>
```

Comment:
It would be better to use this plugin below which is successor 

```
<groupId>org.apache.rat</groupId>
<artifactId>apache-rat-plugin</artifactId>
```